### PR TITLE
Improve Education Timeline #1146

### PR DESCRIPTION
**Pull request by Jyoti6828**
### Screenshots:

image

image

image

### CODE REVIEW

**Comment by smaranjitghose** [comment]
Nice work @Jyoti6828 

## Feedback:

sa

- There should be sufficient space between the card border and the text
- The title(i.e. the name of the school) and the year should have sufficient vertical space from the bullet points below.
- The background color of the cards should be adjusted for dark mode as well

File: education.html [hunk 1]
```diff
@@ -307,17 +308,27 @@ <h6>Masters of Science, Computer Science & Engineering</h6>
...
-        <div class="date">2017-2021</div>
-        <div class="alignment">
-            <h3>Massachusetts Institute of Technology</h3>
+        
+       <!-- <div class="date">2017-2021</div>
+        <div>
+            <h2>Massachusetts Institute of Technology</h3>
             <h6>Bachelors of Science, Computer Science & Engineering</h6>
             <li>Top 1% in the department</li>
             <li>Best undergraduate Reseracher</li>
```
Comment (smaranjitghose) [inline education.html:319]: Remove unused code pleae

File: education.html [hunk 2]
```diff
@@ -307,17 +308,27 @@ <h6>Masters of Science, Computer Science & Engineering</h6>
...
             <li>Betsy Schumacker Woman</li>
             <li>Athlete of the Year Award</li>
+        </div>-->
+		<div class="timeline-img-header" style="background-image: url('assets/images/education-page/edTimeline1.png'); background-repeat: no-repeat; background-size: auto;"></div>
+        <div class="date">2021-2022</div>
+        <div class ="align-items-centern">
+            <h2>Massachusetts Institute</h2>
+            <h6>Masters of Science, Computer Science & Engineering</h6>
+            <li>Top 1% in the department</li>
```
Comment (smaranjitghose) [inline education.html:328]: Try to ensure this data is inside a JSON object as done for other cards in the website to make it more easy to edit for the end user

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

File: education.html (hunk 1 above)
Comment (Jyoti6828) [reply education.html:319]: Ok, I will do this.

File: education.html (hunk 2 above)
Comment (Jyoti6828) [reply education.html:328]: I will try to do that for sure.

**Comment by Jyoti6828** [comment]
image

Is it look good?

**Review by anushbhatia — COMMENTED** [review]
What happened with the navbar

**Comment by Jyoti6828** [comment]
When the images in the timeline are shown the navbar is disturbed. I will create a new issue regarding that is that ok?

**Comment by anushbhatia** [comment]
> When the images in the timeline are shown the navbar is disturbed. I will create a new issue regarding that is that ok? because to solve this issue deadline is passed.

no issues take your time we can extend the deadline for it.

**Comment by Jyoti6828** [comment]
Okay Sure

**Comment by anushbhatia** [comment]
Any update @Jyoti6828

**Comment by Jyoti6828** [comment]
Actually, my exams are starting from Monday onwards  I forget to update you. But I unassign myself from this issue. Sorry for my inconvenience.

**Pull request closed without merging**